src/_api/index.js

```diff
@@ -31,7 +31,7 @@ if (axiosPrivate.interceptors.request.handlers.length === 0) {
   axiosPrivate.interceptors.request.use(
     (config) => {
       if (!config.headers.Authorization) {
-        config.headers.Authorization = `Bearer ${JSON.parse(localStorage.getItem("auth"))?.accessToken}`;
+        config.headers.Authorization = `Bearer ${JSON.parse(localStorage.getItem("auth"))?.token.accessToken}`;
```

Comment:
Please test it in incognito tab where we don't have any prior values in local storage then it should not break

Reply:
It tested as you specified , It did not break